src/app/forms-builder/components/forms-builder.component.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  AfterViewInit,
+  Component,
+  ViewChild,
+  ChangeDetectionStrategy,
+  OnInit,
+  ChangeDetectorRef
+} from '@angular/core';
+import { Store } from '@ngrx/store';
+import { select } from '@ngrx/store';
+import { fromEvent, Observable } from 'rxjs';
+import { map, takeUntil, tap } from 'rxjs/operators';
+import { Router } from '@angular/router';
+
+import { FormsBuilderAccordionComponent } from './forsms-builder-accordion/forms-builder-accordion.component';
+import { selectAccordion, selectDragArea, selectDropArea } from '../../core/state/selectors';
+import { FormsBuilderContentState } from '../../core/state/reducers';
+import { AccordionChangeStylingAction } from '../../core/state/actions/accordionActions';
+import { FormsBuilderService } from '../services/forms-builder.service';
+import { NavbarComponent } from '../../core/navbar/navbar.component';
+
+
+@Component({
+  selector: 'app-forms-builder',
+  templateUrl: './forms-builder.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./forms-builder.component.scss']
+})
+
+export class FormsBuilderComponent implements OnInit,AfterViewInit{
+
+  @ViewChild(FormsBuilderAccordionComponent) accordionComponent:FormsBuilderAccordionComponent
+  @ViewChild(NavbarComponent) navbarComponent:NavbarComponent
+
+  //State
+  public accordionState$:Observable<any>
+  public dragAreaState$:Observable<any>
+  public dropAreaState$:Observable<any>
+
+  public logoutEvent$:Observable<any>
+
+
+  constructor(private store$:Store<FormsBuilderContentState>, private router: Router, private cdr: ChangeDetectorRef, private formsBuilderService: FormsBuilderService) {}
+
+  ngOnInit() {
+    //Detect changes explicitly when app start
+    setTimeout(()=>{
+      this.cdr.detectChanges()
+    },300)
+    //Fetch state data
+    this.accordionState$ = this.store$.pipe(select(selectAccordion))
+    this.dragAreaState$ = this.store$.pipe(select(selectDragArea))
+    this.dropAreaState$ = this.store$.pipe(select(selectDropArea))
+  }
+
+  ngAfterViewInit() {
+    this.logoutEvent$ = fromEvent(this.navbarComponent.logoutBtn._elementRef.nativeElement,'click')
```

Comment:
if you want to know, when user loggedin/out, you can use store for this purpose. on userAuth you can add userInfo to store and if there is user data in the store, that means, the user logged in, if there is no data, user logged out.
On Logout click you can simply create userLogoutAction, which will clean user Info at the store.